lib/pa_ess/updater.ex

```diff
@@ -110,6 +112,7 @@ defmodule PaEss.Updater do
                audio_zones: audio_zones,
                audio_data: [Base.encode64(file)],
                expiration: 30,
+               priority: priority,
```

Comment:
Would it make sense for the message format here to be a struct instead of a map? That would prevent (/would have prevented?) things like not including a `priority` field when we expect one.

Reply:
Mm, if this is ultimately the data we're sending directly out of the app to Scully, then it's probably not that important for it to be a struct. I wasn't sure if we did anything with it within RTS, before that.